server/container/guice/memory-guice/src/test/java/org/apache/james/GuiceJamesServerTest.java

```diff
@@ -109,7 +112,9 @@ public void serverShouldNotBeStartedOnUncaughtException() throws Exception {
 
             try {
                 overWrittenServer.start();
-            } catch (RuntimeException e) {}
+            } catch (RuntimeException e) {
+                LOGGER.info("Ignored expected exception", e);
```

Comment:
I can not as the exception thrown would prevent latest test instructions to be executed.